woodwork/tests/accessor/test_column_accessor.py

```diff
@@ -166,7 +166,8 @@ def test_error_accessing_methods_before_init(sample_series):
         'remove_semantic_tags': [{'new_tag'}],
         'reset_semantic_tags': None,
         'set_logical_type': ['Integer'],
-        'set_semantic_tags': [{'new_tag'}]
+        'set_semantic_tags': [{'new_tag'}],
+        'box_plot_dict': [{}],
```

Comment:
Doesn't matter much, but can this just be `'box_plot_dict': None,`?

Reply:
yep, definitely can!